### PR TITLE
ocaml-basics.0.1.0 - via opam-publish

### PR DESCRIPTION
**Pull request by antoine1fr**
Implements common functionnal patterns / abstractions

---
* Homepage: http://github.com/advanced-schema/ocaml-basics
* Source repo: https://github.com/advanced-schema/ocaml-basics.git
* Bug tracker: http://github.com/advanced-schema/ocaml-basics/issues

---
### opam-lint failures
- **WARNING** 97 long description unspecified

---

Pull-request generated by opam-publish v0.3.4

### CODE REVIEW

File: packages/ocaml-basics/ocaml-basics.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  [make]
+]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "ocaml-basics"]
+depends: [
+  "ocamlfind" {build}
+  "oasis" {>= "0.4"}
+  "result" {>= "1.2"}
+  "ppx_sexp_conv" {>= "0.9"}
+  "sexplib" {>= "0.9"}
```
Comment (avsm) [inline packages/ocaml-basics/ocaml-basics.0.1.0/opam]: i think this should be `>="v0.9"`? (note the "v" in the version number)

Comment (antoine1fr) [inline packages/ocaml-basics/ocaml-basics.0.1.0/opam]: Ok I'll fix that. I was a bit lost regarding sexplib's version scheme. :)

Comment (avsm) [inline packages/ocaml-basics/ocaml-basics.0.1.0/opam]: this one's a bit non-obvious -- perhaps we should put a note into `opam-repository/CHANGES` @diml? explaining the cunning reasoning behind the `v`? :-)

Comment (antoine1fr) [inline packages/ocaml-basics/ocaml-basics.0.1.0/opam]: Is it some sort of a hack with the ASCII comparison? ('v' > '113')

Comment (antoine1fr) [inline packages/ocaml-basics/ocaml-basics.0.1.0/opam]: @avsm I've taken into account your past comment. Do you have any other comment? :)

**Comment by avsm** [comment]
thanks for the changes!

File: packages/ocaml-basics/ocaml-basics.0.1.0/opam (hunk 1 above)
Comment (diml) [inline packages/ocaml-basics/ocaml-basics.0.1.0/opam]: Ok, i'll add a note. @antoine1fr that's exactly that, we wanted to switch to a more classic versioning scheme but had to keep versions going up

**Pull request merged**